agent/nmap_options.py

```diff
@@ -67,5 +74,5 @@ def command_options(self) -> List[str]:
         command_options.extend(self._set_dns_resolution_option())
         command_options.extend(self._set_ports_option())
         command_options.extend(self._set_timing_option())
+        command_options.extend(self._set_script())
```

Comment:
This is all missing unit tests!!!!!

Reply:
when you approve the logic , i will move to unit tests